software/apisix.yml

```
@@ -0,0 +1,18 @@
+name: Apache APISIX
+website_url: http://apisix.apache.org/
+description: The Most Popular Open Source Cloud-Native API Gateway
```

A better explanation.
```suggestion
description: A dynamic, real-time, high-performance API Gateway.
```